modules/series-service-impl/src/main/java/org/opencastproject/series/impl/persistence/SeriesServiceDatabaseImpl.java

```diff
@@ -534,13 +610,14 @@ public boolean storeSeriesAccessControl(String seriesId, AccessControlList acces
         updated = true;
       }
       entity.setAccessControl(serializedAC);
+      entity.setModifiedDate(new Date());
       em.merge(entity);
       tx.commit();
       return updated;
     } catch (NotFoundException e) {
       throw e;
     } catch (Exception e) {
-      logger.error("Could not update series: {}", e.getMessage());
+      logger.error("Could not store ACL for series '{}': {}", seriesId, e.getMessage());
```

Comment:
Do you want me to change this for all these cases in the file? Basically every existing method first logs an error, then throws the exception. I can if you want, but it's completely unrelated to the rest of this PR and it might be better to immediately create such a PR against the community version.